lab_1_keywords_tfidf/main.py

```diff
@@ -66,7 +87,24 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (isinstance(frequencies, dict) and isinstance(top, int)
+            and top > 0 and frequencies and isinstance(top, bool) is False):
```

Comment:
to check whether the condition is false, we use `not` instead of `is False`

Reply:
I don't understand what do you mean:( **is False** - is a construction that required to pass the Style code check. No another construction isn't suitable to pass this check.

Reply:
ok i've done it